src/TDebug.cpp

```diff
@@ -25,22 +25,67 @@
 #include "TConsole.h"
 #include "mudlet.h"
 
+// This is a Unicode NON-character code which is explicitly undisplayable but
+// can be embedded for our own internal purposes - in this case to flag that
+// the text is a continuation of a previous one and should NOT have a marking
+// prepended to indicate from which profile it came.
+const QChar TDebug::csmContinue = QChar(0xFFFF);
+
+QMap<const Host*, QPair<QString, QChar>> TDebug::smIdentifierMap;
+QSet<QChar> TDebug::smUsedIdentifiers;
+QQueue<TDebugMessage> TDebug::smMessageQueue;
 
 TDebug::TDebug(const QColor& c, const QColor& d)
-: msg()
-, fgColor(c)
+: fgColor(c)
 , bgColor(d)
 {
 }
 
-// code is a dummy variable so that using operator>>() (at the end of a series
-// of operator<<() calls) causes the accumulated message to be printed/flushed
-// out...
-TDebug& TDebug::operator>>(const int code)
+// This is the method that pushes the accumulated text out to the Central Debug
+// Console. If the accumulated message text (in msg) begins with csmContinue
+// then this message is appended onto a previous one and should NOT include
+// any indicator (at the beginning) of the profile from which it came - which
+// is now deduced from the supplied Host pointer.
+TDebug& TDebug::operator>>(Host* pHost)
```

Comment:
This in part repeats a lengthy comment from 10 lines prior, please separate causes or pull together in one.

Reply:
:mag_right: The comments do server different purposes but I agree there is some redundancy in this text. I will see what I can eliminate...